packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -166,5 +168,10 @@ internal class DatadogSDKWrapper : DatadogWrapper {
     override fun clearAllData() {
         return Datadog.clearAllData(DatadogSDKWrapperStorage.getSdkCore())
     }
+
+    internal companion object {
+        internal const val RUM_ENABLE_LOGS_DEFAULT = true
+        internal const val TRACES_ENABLE_LOGS_DEFAULT = true
```

Comment:
I think better naming would be something like `BUNDLE_LOGS_WITH_RUM_DEFAULT` / `BUNDLE_LOGS_WITH_TRACES_DEFAULT`: this has more consistency and have a clear direction what goes there (naming like `TRACES_ENABLE_LOGS` is not clear enough: are we enable some logging in traces or tracing in logging?)